lib/single-new-eks-fargate-opensource-observability-pattern/index.ts

```diff
@@ -98,7 +98,7 @@ export default class SingleNewEksFargateOpenSourceObservabilityConstruct {
         const addOns: Array<blueprints.ClusterAddOn> = [
             new blueprints.addons.VpcCniAddOn(),
             new blueprints.addons.CoreDnsAddOn({
-                version: "v1.10.1-eksbuild.1",
+                version: "auto",
```

Comment:
this is inaccurate. We dont support auto with individual addons. You have to add the latest supported version,